build-tools/packages/build-tools/src/typeValidator/typeData.ts

```diff
@@ -3,23 +3,26 @@
  * Licensed under the MIT License.
  */
 
-import { Node, ts } from "ts-morph";
+import { Node } from "ts-morph";
 
 export interface TypeData {
 	/**
 	 * Includes namespace prefix if needed.
 	 */
 	readonly name: string;
-	readonly kind: string;
+	/**
+	 * Test case name.
+	 * Uniquely identifies a test case with a string safe to use as an identifier.
+	 * Consists of a non-colliding "mangled" identifier for the type.
+	 */
+	readonly testCaseName: string;
 	readonly node: Node;
 	readonly tags: ReadonlySet<string>;
-}
-
-/**
- * Creates a non-colliding "mangled" identifier for the type.
- */
-export function getFullTypeName(typeData: TypeData) {
-	return `${typeData.kind}_${typeData.name.replace(/\./g, "_")}`;
+	/**
+	 * Does this TypeData to the the name item, or the typeof the name item.
```

Comment:
I think you're missing a verb here but I don't know which one. "belong"?

Reply:
```suggestion
	 * Indicates if this TypeData refer to the named item (false), or the typeof the named item (true).
```